projects/hslayers/src/components/add-layers/add-layers.component.ts

```diff
@@ -73,13 +76,13 @@ export class HsAddLayersComponent {
   }
 
   /**
-   * Change detail panel template according to selected type
-   *
    * @function templateByType
-   * @returns {string} template Path to correct type template
+   * @description Change detail panel template according to selected type
+   * @todo unused
+   * @returns {string} Path to correct type template
    */
-  templateByType(): string {
```

Comment:
This method is no longer used. I intended to delete it, but obviously forgot it in the end.